Procfile

```diff
@@ -1 +1,5 @@
 web: npm run start:server:prod
+scale4: npm run heroku:scale 4
+scale5: npm run heroku:scale 5
+scale6: npm run heroku:scale 6
+scale7: npm run heroku:scale 7
```

Comment:
As far as I can tell from reading [the docs](https://devcenter.heroku.com/articles/procfile):

1. We need a Procfile entry to schedule a job because the scheduled job runs on a separate dyno. We can't directly schedule `npm run heroku:scale 5`
2. We can use `$VAR` to refer to env vars we've defined, but I don't think there is a way to pass in a variable to a command in the Procfile when we invoke it

so I think the best we can do here is give ourselves some "presets" we can use in schedule jobs.